cartographer_ros/cartographer_ros/cartographer_grpc/node_grpc_main.cc

```diff
@@ -57,7 +57,7 @@ void Run() {
   Node node(node_options, std::move(map_builder), &tf_buffer);
 
   if (!FLAGS_map_filename.empty()) {
```

Comment:
Rename the flag as well, to be consistent with the offline and online node?

Reply:
Or, maybe we can delay that rename once GRPC supports loading unfrozen trajectories as well.